compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/XMLStepIndexedExtendNode.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
are the years manually updated? You need to add an entry to `syntax_node_metadata.json`. It will automatically pick the year from there.